docs/reference/verify-assets.mdx

```diff
@@ -106,11 +109,68 @@ As the SBOM was signed with our private key, you can now inspect it to track dow
 
 ## Verify image-based gadgets
 
-### With `ig`
-
 Like our container image, we sign all our image-based gadgets.
-The signature are verified by default using Inspektor Gadget public key:
+The signature are verified by default using Inspektor Gadget public key.
+
+<Tabs groupId="env">
+<TabItem value="kubectl-gadget" label="kubectl gadget">
+Compared to `ig`, you cannot specify information with regard to verification when calling `kubectl gadget run`:
+
+```bash
+$ kubectl gadget run --public-keys="$(cat your-key.pub)" trace_exec
+Error: unknown flag: --public-keys
+$ kubectl gadget run --verify-image=false trace_exec
+Error: unknown flag: --verify-image
+```
+
+Instead, all these information are set once at deploy time.
+This avoid them being tampered by any users.
+By default all image-based gadgets are verified using Inspektor Gadget public key:
```

Comment:
```suggestion
When using the Inspektor Gadget on Kubernetes, you have to configure these features at deploy time. This enforces the policies so that users cannot override them.
By default all image-based gadgets are verified using Inspektor Gadget public key:
```